content/docs/engine-dev/tools/project-manager/_index.md

```diff
@@ -1,13 +1,160 @@
 ---
 linkTitle: Project Manager
-title: Project Manager Developer Documentation
-description: Developer documentation for the Project Manager tool.
+title: Project Manager Developer High Level Overview (Boot-up sequence)
+description: High Level Developer Overview for the Project Manager tool.
 ---
 
 {{< note >}}
 This information is for developers of the **Project Manager** tool. If you're a user working with Project Manager, please refer to the [Project Manager User Guide](/docs/user-guide/project-config/project-manager).
 {{< /note >}}
 
-{{< todo >}}
-This is a stub article intended to serve as an example of guide structure, and is not necessarily content that will be made available in the near future.
-{{< /todo >}}
+
+```mermaid
```

Comment:
Make a note of the links in the graph nodes to increase awareness of their availability.
```suggestion
{{< tip >}}
Click on a node to open the relevant source code for that node.
{{< /tip >}}

```mermaid
```